lessons/04-client-jquery-ajax/README.md

```diff
@@ -1,113 +1,35 @@
-A useful shortcut here is the magnifying glass in the top-left corner of the console.
-Click that and then click in the browser somewhere to jump to the HTML of that element.
-* Network: While you're in this tab, refresh the browser.
-The tab will list all the requests that the web page makes, including the size, timing, and specific response.
-This can be useful for debugging your server's responses or other API's.
-* Sources: The holy grail of JavaScript debugging tools. Set breakpoints, step through code,
-all the fun stuff that an IDE would give you. We'll get back to this when we go over debugging in more depth.
-* Console: A full JavaScript console that operates in the current scope of the Javascript on the page (including inside breakpoints).
-Run commands, and read through your client-side `console.logs` in this window. **Server-side javascript `console.log` statements will log in your terminal. When you have `console.log` statements in client-side javascript (which we'll discuss more below), they will show up in your developer console.** 
-As a bonus, if you click the "console icon" ![console icon](images/consoleicon.png) in the upper-right corner,
-the console will pop up below whatever other tab you want to reference.
-
-###Javascript Code in Console
-With the developer console, you can actually interface and inject your own javascript into a webpage. If you open up the console (as described in last bullet point above) you can type in your own commands and have them execute. Try typing in:
-```
-alert('hi there')
-```
-
-Not only that, you can also see their javascript files and even have access to any variables. Where you find those is under the sources tab in the console. Follow along and pull up their files as shown in the screenshot: 
+# Lesson 4 - The Client, jQuery, and AJAX
+Web Applications handle much of their database communication and logic on the server, but the front-end is what the client actually sees and interacts with. Without a clean, intuitive, responsive, and enjoyable client-side, the best server application in the world wouldn't be much.
 
-![Cookie Clicker](images/cookie1.png)
+Today we are going to learn the basics of building and manipulating the client and building communication routes between the client and server.
 
-If you investigate (or just guess) you will notice a variable called "Game" and if you start to type that into the console it will even suggest an autocomplete for it. You can look at its functions and realize that there is a variable called cookiePs, which you might guess corresponds to cookies per second. You can set cookiesPs to any value and see what happens:
-
-![Cookie Clicker hacked](images/cookie2.png)
-
-The takeaway from all this is that the client is NOT secure. A smart user can take your app apart on the client and do anything he wants with it. All security needs to be implemented on the server side so that it cannot be meddled with. 
+## README Table of Contents
```

Comment:
Can we have the table of contents also include a link to the in-class and the homework?